packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsCore.tsx

```diff
@@ -146,17 +147,24 @@ export class SuggestionsCore<T> extends BaseComponent<ISuggestionsCoreProps<T>,
             role="listitem"
             aria-label={suggestion.ariaLabel}
           >
-            <TypedSuggestionsItem
-              id={'sug-item' + index}
-              suggestionModel={suggestion}
-              // tslint:disable-next-line:no-any
-              RenderSuggestion={onRenderSuggestion as any}
-              onClick={this._onClickTypedSuggestionsItem(suggestion.item, index)}
-              className={suggestionsItemClassName}
-              showRemoveButton={showRemoveButtons}
-              onRemoveItem={this._onRemoveTypedSuggestionsItem(suggestion.item, index)}
-              isSelectedOverride={index === this.currentIndex}
-            />
+            {renderSuggestionsItemAsDiv && onRenderSuggestion ? (
+              onRenderSuggestion(suggestion.item, {
+                suggestionModel: suggestion,
+                onClick: this._onClickTypedSuggestionsItem(suggestion.item, index)
+              })
+            ) : (
+              <TypedSuggestionsItem
```

Comment:
I'm pretty sure the pickers have already been changed so that it will render as a div instead of a button, I don't think this change is necessary.

Reply:
Correct me if I m wrong, the suggestionsItem still has a command button wrapping the content in this line: https://github.com/OfficeDev/office-ui-fabric-react/blob/1b9405b3b64839975e16ee1fad04d7868d9e3b99/packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx#L41

Reply:
If you don't want any buttons then this change should be in the suggestions item to allow you to replace the button.

Reply:
@joschect So you mean pass the props one level down?